src/components/Form/Form.test.js

```diff
@@ -0,0 +1,19 @@
+import { render, screen } from '@testing-library/react';
+import Form from './Form';
+import { translations } from '../../utils/translations';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
Because I have error:
![image](https://user-images.githubusercontent.com/69632427/171386159-40106f65-8b31-4474-81b5-71ef35d5b58e.png)


So to fit it I need this:
![image](https://user-images.githubusercontent.com/69632427/171385953-855466fc-0519-4908-8fc6-ed87cd974f2c.png)

Reply:
usually it's enough to import jest-dom once in some file like src/setupTests.js and define this setup in [config](https://jestjs.io/docs/configuration#setupfiles-array) if needed

Reply:
Thank you for advice. Fixed and works fine 👍